site/content/blogs/release-v118.en.md

```diff
@@ -21,15 +21,27 @@ enabling you to focus on developing your application, instead of writing deploym
 
 See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
-## Certificate import
+## Certificate Import
 _Contributed by [Penghao He](https://github.com/iamhopaul123/)_
 
 ## Ordering Deployments in a Pipeline
 _Contributed by [Efe Karakus](https://github.com/efekarakus/)_
 
-## "recreate" Strategy for Faster Deployments
+## "recreate" Strategy for Faster Redeployments
 _Contributed by [Parag Bhingre](https://github.com/paragbhingre/)_
 
+!!!alert
+    Please do not set "recreate" for any services that are in production stage because of the deployment downtime.
+
+In order to support faster redeployments in developing stage, users can specify "recreate" as the deployment strategy in the service manifest:
```

Comment:
What do you think of describing first "what we do today" and then "what we now offer". I actually wrote a bunch so that you can edit based on it but I lost it. So, so, so sad...

Something like "Today, a service deployment will first spin up new tasks, wait for them to be stable, and only after that will it stop the old tasks. With recreate, it will first stop the old tasks and then spin up new ones. ".